packages/code/CodeCommand.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+var AnnotationCommand = require('../../ui/AnnotationCommand');
+var oo = require('../../util/oo');
+
+function CodeCommand() {
+  AnnotationCommand.apply(this, arguments);
+}
+oo.inherit(CodeCommand, AnnotationCommand);
+
+CodeCommand.static = {
```

Comment:
This is may cause problems as you loose the prototype chain of the static object. I.e. you loose static inheritance.